### PR TITLE
Add updated Attachments page; update associated pages

### PR DESCRIPTION
**Pull request by PeloWriter**

### CODE REVIEW

File: src/collections/_documentation/enriching-error-data/attachments.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Attachments persist for 30 days; if your total storage included in your quota is exceeded, attachments will not be stored. You can delete attachments or their containing events at any time. Deleting an attachment does not affect your quota - Sentry counts an attachment toward your quota as soon as it is stored. 
+
+Learn more about how attachments impact your [quota]({%- link _documentation/accounts/quotas/index.md %}).
+
+## Crash Reports and Privacy
+
+Crash reports may contain sensitive information about the target system, such as environment variables, local pathnames, or in-memory representations of input fields, including passwords. By default, Sentry only uses crash report files to create events and immediately drops them. All sensitive information is stripped from the resulting events.
+
+All other types of attachments, such as log files or screen shots, are stored for 30 days when they are sent to Sentry. Note that Sentry does not apply data scrubbing to attachments.
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/attachments.md:25]: Screenshot is one word. 😸

Comment (MimiDumpling) [reply src/collections/_documentation/enriching-error-data/attachments.md:25]: "...30 days when they are sent..." can be shortened to "...30 days when sent..."

File: src/collections/_documentation/enriching-error-data/attachments.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You can enable *Store Native Crash Reports* in your organization's **Security and Privacy Settings**. By default, this setting is disabled. Determine the maximum number of crash reports that will be stored per issue; disabled, unlimited, or maximum per issue:
+
+[{% asset attachments/slide-bar-native-crashes.png alt="Attachments Example" %}]({% asset attachments/slide-bar-native-crashes.png @path %})
+
+If you set a limit per issue, as in the example above, a limit of 10, Sentry will store the first 10 attachments associated with this issue, but drop any that follow. To make room for additional attachments, delete them. Sentry will then accept attachments until the limit is reached again.
+
+### Access to Attachments
+
+To limit access to attachments, navigate to your organization's **General Settings**, then select the *Attachments Access* dropdown to set appropriate access—any member of your organization, the organization billing owner, member, admin, manager, or owner.
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/attachments.md:37]: spacing before and after the em dash in "access-any"

File: src/collections/_documentation/enriching-error-data/attachments.md (hunk 1 above)
Comment (PeloWriter) [reply src/collections/_documentation/enriching-error-data/attachments.md:25]: Thanks.

File: src/collections/_documentation/enriching-error-data/attachments.md (hunk 2 above)
Comment (PeloWriter) [reply src/collections/_documentation/enriching-error-data/attachments.md:37]: Ok. But we should discuss as a standard - em dashes absorb a ton of space; additional space seems superfluous (to me).

**Comment by jan-auer** [comment]
@PeloWriter I just noticed that we will also have to update this document, then we got them all:
https://docs.sentry.io/platforms/native/ue4/

**Pull request merged**